standalone/CMakeLists.txt

```diff
@@ -15,11 +17,11 @@ include(../cmake/options.cmake)
 
 # ---- Dependencies ----
 
-# FIXME: EXCLUDE_FROM_ALL must be off or fmt will not be installed! CK
+# NOTE: EXCLUDE_FROM_ALL must be off or cxxopts target will NOT be installed! CK
```

Comment:
Same as above: `has to be` instead of `must be`.

Maybe you could also add a note that this is needed because you aren't using the short form for specifying the repository (which would implicitly set EXCLUDE_FROM_ALL to YES).